python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,52 @@
+from ray._private.structured_logging.formatters import TextFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+from typing import Union
+
+LOG_MODE_DICT = {
+    "TEXT": lambda log_level: {
+        "version": 1,
+        "disable_existing_loggers": False,
+        "formatters": {
+            "text": {
+                "()": TextFormatter,
+            },
+        },
+        "filters": {
+            "core_context": {
+                "()": CoreContextFilter,
+            },
+        },
+        "handlers": {
+            "console": {
+                "level": log_level,
+                "class": "logging.StreamHandler",
+                "formatter": "text",
+                "filters": ["core_context"],
+            },
+        },
+        "root": {
+            "level": log_level,
+            "handlers": ["console"],
+        },
+    },
+}
+
+
+class LoggingConfig:
+    def __init__(self, log_config: Union[dict, str] = "TEXT", log_level: str = "INFO"):
+        self.log_config = log_config
+        self.log_level = log_level
```

Comment:
Let's add docstring since it's public API

Reply:
Updated https://github.com/ray-project/ray/pull/45344/commits/69f8b57dc5290ed2e5343a5962fa87f79e1b270d